test/e2e/apimachinery/webhook.go

```diff
@@ -36,11 +37,15 @@ import (
 )
 
 const (
-	secretName        = "sample-webhook-secret"
-	deploymentName    = "sample-webhook-deployment"
-	serviceName       = "e2e-test-webhook"
-	roleBindingName   = "webhook-auth-reader"
-	webhookConfigName = "e2e-test-webhook-config"
+	secretName                   = "sample-webhook-secret"
+	deploymentName               = "sample-webhook-deployment"
+	serviceName                  = "e2e-test-webhook"
+	roleBindingName              = "webhook-auth-reader"
+	webhookConfigName            = "e2e-test-webhook-config"
+	whitelistNamespaceLabelKey   = "skip-webhook-admission"
```

Comment:
I think whitelist/blacklist are ambiguous in this situation, since it's not clear if the list is namespaces that the webhook checks or ones it doesn't check. Maybe just s/whitelist/skip/.

Reply:
Done.